packages/editor/README.md

```diff
@@ -128,14 +128,30 @@ Example:
 
 ### AutosaveMonitor
 
-AutosaveMonitor component. Monitors the changes made to the edited post and triggers autosave if necessary.
+Monitors the changes made to the edited post and triggers autosave if necessary.
+
+The logic is straightforward: a check is performed every `props.interval` seconds. If any changes are detected, `props.autosave()` is called. The time between the change and the autosave varies but is no larger than `props.interval` seconds. Refer to the code below for more details, such as the specific way of detecting changes.
+
+There are two caveats:
+
+-   If `props.isAutosaveable` happens to be false at a time of checking for changes, the check is retried every second.
+-   The timer may be disabled by setting `props.disableIntervalChecks` to `true`. In that mode, any change will immediately trigger `props.autosave()`.
 
 _Usage_
 
 ```jsx
 <AutosaveMonitor interval={ 30000 } />
 ```
 
+_Parameters_
+
+-   _props_ `Object`: - The properties passed to the component.
```

Comment:
The static analysis job failure of the CI is valid and is about:

>15:2  error  Expected JSDoc block lines to be aligned                                      jsdoc/check-line-alignment
[2]   18:1  error  Invalid JSDoc tag (preference). Replace "constructor" JSDoc tag with "class"  jsdoc/check-tag-names
[2]   94:1  error  Expected JSDoc block lines to be aligned                                      jsdoc/check-line-alignment

If you eventually remove the above duplicate description of props, you just need to align the comments of the props here.

You can check [this doc's section](https://github.com/WordPress/gutenberg/blob/trunk/docs/contributors/code/getting-started-with-code-contribution.md#eslint) about setting this up in your code editor.